ddtrace/bootstrap/sitecustomize.py

```diff
@@ -88,7 +90,8 @@ def update_patched_modules():
 
     opts = {}  # type: Dict[str, Any]
 
-    if asbool(os.environ.get("DATADOG_TRACE_ENABLED", True)):
+    dd_trace_enabled = get_env("trace", "enabled")
```

Comment:
```suggestion
    dd_trace_enabled = get_env("trace", "enabled", default=True)
```

Reply:
I'm going to mark this suggestion as resolved as the latest commit I tried does use `dd_trace_enabled = get_env("trace", "enabled", default=True)` and this did fix the error I was getting with some integrations not being patched.